recipes/vscode-jupyter/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
```

Comment:
The latest release of vscode-jupyter ([2021.02.1](https://github.com/microsoft/vscode-jupyter/releases/tag/2021.02.1)) requires vscode `^1.53.0` which `code-server` does not support yet ([open PR](https://github.com/cdr/code-server/pull/2773)).